src/components/SlateEditor/plugins/file/index.js

```diff
@@ -12,10 +12,16 @@ import Filelist from './Filelist';
 export default () => {
   const schema = {
     document: {},
+    blocks: {
+      file: {
+        isVoid: true,
+        data: {},
+      },
+    },
   };
 
   /* eslint-disable react/prop-types */
-  const renderNode = (props, editor, next) => {
+  const renderNode = (props, next) => {
```

Comment:
Nå vil next være satt til editor, og applikasjonen kræsjer for plugins etter File-plugin.

Reply:
sry